x-pack/docs/en/ilm/apis/explain.asciidoc

```diff
@@ -0,0 +1,116 @@
+[role="xpack"]
+[testenv="basic"]
+[[ilm-explain]]
+=== Explain Lifecycle API
+++++
+<titleabbrev>Explain Lifecycle</titleabbrev>
+++++
+
+Shows the current lifecycle staus for an index.
+
+==== Request
+
+`GET <index>/_ilm/explain`
+
+==== Description
+
+This API returns information relating to the current lifecycle state of an
+index. This includes information such as the currently executing phase, action
+and step and the timestamp when the index entered them. It also shows the
+version of the current phase that is being run and in the event that there has
```

Comment:
I think the "version" of the current phase is a little confusing, I would maybe change it to say the definition of the current phase that the index is in (not being run).

Also, `being run and in` -> `being run and, in`